### PR TITLE
Page layout

### PR DESCRIPTION
**Pull request by cindy1u0**
Header + mobile style 
Click on preview on vercel bot you can see what it looks like

### CODE REVIEW

File: frontend/src/components/Nav.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    };
+    window.addEventListener("resize", resizeScreen);
+    return () => {
+      window.removeEventListener("resize", resizeScreen);
+    };
+  }, []);
+
+  const HeaderItems = (s) => {
+    return (
+      <>
```
Comment (jrosen081) [inline frontend/src/components/Nav.js:95]: What does this do? I've never seen it before.

**Review by jrosen081 — APPROVED** [review]
This looks amazing! Love that we still have Da-Jin on the website

**Comment by cindy1u0** [comment]
HAHAHAH dajin's our bRaNd

File: frontend/src/components/Nav.js (hunk 1 above)
Comment (cindy1u0) [reply frontend/src/components/Nav.js:95]: Ohh it's like when you don't wanna make another parent div you just use an empty <></> to wrap around all your child components

**Pull request merged**